src/OSPSuite.Assets.Images/ApplicationIcon.cs

```diff
@@ -31,20 +31,14 @@ private static SvgImage bytesToImage(byte[] bytes)
 
       public static implicit operator SvgImage(ApplicationIcon icon) => icon.ToSvgImage();
 
+      public static implicit operator Image(ApplicationIcon icon) => icon.ToImage();
```

Comment:
@Yuri05 When we moved to svg, we lost the ability to convert implicitly to image. This was a breaking change that I did nto realize. The component that display in the grid only accepts image and hence the bug. Thsi is also a bug in a few other places in PKSim where we are returning an icon instead of an image (pop import from csv file for instance). Hence I decided to fix it here instead of everywhere